plugins/woocommerce/includes/admin/meta-boxes/class-wc-meta-box-product-data.php

```diff
@@ -414,6 +414,9 @@ public static function save( $post_id, $post ) {
 			WC_Admin_Meta_Boxes::add_error( $errors->get_error_message() );
 		}
 
+		// Remove _product_template_id for products that were created with the new product editor.
+		$product->delete_meta_data( '_product_template_id' );
+
```

Comment:
Do we want to wrap this in a if statement to check if the product block editor is enabled or not?
Or does this `php` file only get rendered on the classic experience anyway?

Reply:
This code is part of the classic editor and only runs when saving in the classic editor. Maybe the comment gave you the wrong idea.

This will only run when a product was created with the new product editor, and edited with the classic editor afterwards.